src/common/intl.cpp

```diff
@@ -270,26 +273,35 @@ bool wxLocale::Init(const wxString& name,
                     wxS("no locale to set in wxLocale::Init()") );
     }
 
-    if ( const wxLanguageInfo* langInfo = FindLanguageInfo(szLocale) )
+    int languageId = wxLANGUAGE_UNKNOWN;
+    wxLocaleIdent localeId = wxLocaleIdent::FromTag(szLocale);
+    if ( const wxLanguageInfo* langInfo = wxUILocale::FindLanguageInfo(localeId) )
```

Comment:
Should we check for `!localeId.IsEmpty()` here or is it guaranteed that `FindLanguageInfo()` returns null for it?

Reply:
> Should we check for `!localeId.IsEmpty()` here or is it guaranteed that `FindLanguageInfo()` returns null for it?

Good question. I will check that, and will modify the code, if necessary.